data/json/items/gunmod/underbarrel.json

```
@@ -29,6 +29,79 @@
     "mod_targets": [ "pistol", "smg", "rifle", "shotgun", "launcher", "crossbow" ],
     "extend": { "flags": [ "PUMP_RAIL_COMPATIBLE" ] }
   },
+  {
+    "id": "bipod_handguard",
+    "type": "GUNMOD",
+    "//": "based on stuff like the BLK LBL",
+    "name": { "str": "bipod hand guard" },
+    "description": "Bipods are commonly used on rifles and machine guns to provide a forward rest and reduce motion.  Although they greatly improve handling of recoil, they are usable only on certain surfaces (or from the prone position).  This one is integrated into a high end hand guard, replacing the factory hand guard of your weapon, allowing for comfortable shooting with or without the bipod deployed.  Currently the bipod legs are stowed.",
+    "weight": "400 g",
+    "volume": "500 ml",
+    "integral_volume": "0 ml",
+    "integral_weight": "0 g",
+    "price": 68000,
+    "price_postapoc": 500,
+    "install_time": "20 m",
+    "to_hit": -1,
+    "bashing": 10,
+    "material": [ "aluminum" ],
+    "symbol": ":",
+    "color": "brown",
+    "location": "underbarrel",
+    "mod_targets": [ "rifle" ],
+    "handling_modifier": 4,
+    "use_action": {
+      "type": "transform",
+      "msg": "You extend the bipod legs.",
+      "menu_text": "Fold out",
+      "target": "bipod_handguard_deployed",
+      "moves": 120
+    },
+    "min_skills": [ [ "weapon", 2 ] ]
+  },
+  {
+    "id": "bipod_handguard_deployed",
+    "type": "GUNMOD",
+    "name": { "str": "bipod hand guard (deployed)" },
```

```suggestion
    "name": { "str": "bipod hand guard (deployed)", "str_pl": "bipod hand guards (deployed)" },
```
(I contemplated giving this a meme plural because this shouldn't exist)